design-system/materials/internals/definition.yaml

```diff
@@ -74,10 +79,25 @@ choiceGroupsByTheme:
       prefix: font-size
       choices:
         font-size-m: '1rem'
+        font-size-10: '0.75rem'
+        font-size-20: '0.875rem'
+        font-size-30: '1rem'
+        font-size-40: '1.125rem'
+        font-size-50: '1.25rem'
+        font-size-60: '1.5rem'
+    fontWeight:
+      label: Font weight
+      prefix: font-weight
+      choices:
+        font-weight-400: '400'
+        font-weight-500: '500'
+        font-weight-600: '600'
+        font-weight-700: '700'
```

Comment:
Are these changes relevant to this PR? I thought they are implemented in the typography PR #2317 🤔

Reply:
Buttons have different font weights based on the theme.
In this PR I only need `400` and `500` but I opted for including the other ones as I thought we would need them anyway. I can remove `600` and `700` if you think it's better.